cohort_4/playbook/ACTIVITIES.md

```diff
@@ -689,6 +689,35 @@ Uniqueness is also important - no two people should be paired together again
 unless everyone has already been paired with each other. This will help members
 of the club all form bonds with each other.
 
+## Types of Pairing
+
+### Pair programming between beginner-advanced pairs
+
+Pair programming between beginner and advanced hackers is fairly similar to
+normal pair programming. The most benefit will come from the beginner driving
+and the advanced hacker navigating, so rotating is not as important a factor in
+these situations (hackers should rotate if their bordom becomes a blocking
+factor for them to continue working together, but optimize for the beginner
+driving and the advanced hacker navigating as much as possible).
```

Comment:
They should still be rotating regularly. I believe the problem is that advanced programmers have a tendency to want to drive the coding completely, and we need to get them to resist that urge.